.github/workflows/test_comprehensive.yml

```
@@ -56,12 +56,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest, windows-latest, macos-latest]
+        platform: [ubuntu-latest, windows-latest, macos]
```

```suggestion
        platform: [ubuntu-latest, windows-latest]
```

Sorry meant to delete the full macos, entry not sure what happend.